### PR TITLE
WT-7243 Fix test_bug025, don't expect RTS to always issue an error message

### PR DESCRIPTION
**Pull request by ddanderson**

### CODE REVIEW

**Comment by ddanderson** [comment]
@sueloverso, I think this will fix the error you saw on bug025.

**Comment by ddanderson** [comment]
BTW, the reason we're looking at error messages at all is that in any python test, if there is any stderr or stdout that is not accounted for, the test will fail.

File: test/suite/test_bug025.py [hunk 1]
```diff
@@ -55,31 +55,31 @@ def test_bug025(self):
         pos = os.path.getsize(filename) - 1024
         os.remove(filename)
 
-        # We get an error message, but the open succeeds.
-        with self.expectedStderrPattern('.*No such file or directory'):
+        # We will get error output, but not always in the same API calls from run to run,
+        # in particular the open connection doesn't always report the missing file, as
+        # index files are usually lazily loaded. As long as the missing file is reported
+        # at least once in the following code, it's good.  We're doing a pretty bad thing
+        # by removing the index file, whether we get two messages or three is not important.
```
Comment (sueloverso) [inline test/suite/test_bug025.py:62]: I think the last sentence is unnecessary and a bit confusing about number of messages. The previous sentence sums it up nicely. I suggest removing the last comment sentence.

**Review by sueloverso — APPROVED** [review]
LGTM minor suggestion on the comment.

**Pull request merged**